src/configs/settings.ts

```diff
@@ -702,9 +704,140 @@ export const symbolLayerConfig = {
 };
 
 /** Default colors layer fill colors per administrative level */
-export const adminLayerColors = ['purple', 'blue', 'green', 'yellow', 'orange', 'red'];
+export const adminLayerColors = ['black', 'red', 'orange', 'yellow', 'green'];
 export type adminLayerColorsType = typeof adminLayerColors[number];
 
+export type JurisidictionTypes = Jurisdiction | NamibiaIrsReportingJurisdiction;
+
+export const NamibiaIrsReportingProps = [
+  'foundcoverage',
+  'householdsnotaccessible',
+  'lockedfirst',
+  'lockedmopup',
+  'refusalsfirst',
+  'refusalsmopup',
+  'sprayeffectiveness',
+  'structuresfound',
+  'structuressprayed',
+  'targetcoverage',
+];
+
+export interface NamibiaIrsReportingJurisdiction extends Jurisdiction {
+  foundcoverage: number;
+  householdsnotaccessible: number;
+  lockedfirst: number;
+  lockedmopup: number;
+  refusalsfirst: number;
+  refusalsmopup: number;
+  sprayeffectiveness: number;
+  structuresfound: number;
+  structuressprayed: number;
+  targetcoverage: number;
+}
+export interface IrsReportingCongif {
+  drilldownColumnGetters: {
+    [key: string]: () => any;
+  };
+  juridictionTyper: (j: any) => JurisidictionTypes;
+  sliceProps: string[];
+}
+/* tslint:disable:object-literal-sort-keys */
+export const irsReportingCongif: { [key: string]: IrsReportingCongif } = {
+  '550': {
+    drilldownColumnGetters: {
```

Comment:
`drilldownColumnGetters` are functions returning columns for drilldown table props, this is where we can build in contextually-specific columns (like background-colors)

Reply:
@moshthepitt @p-netm @kahama94 - would love your thoughts / critique of this approach of making the IRS Reporting pages configurable 🙏